suitcase/mongo_normalized/__init__.py

```diff
@@ -130,16 +126,40 @@ def update(self, name, doc):
                 f"Only updates to 'start' documents are supported.")
 
     def start(self, doc):
-        self._run_start_collection.insert_one(doc)
+        try:
```

Comment:
Could this be refactored into a function that takes in a `doc` and a mongo collection object, to avoid repetition?

Reply:
Sure ill do that